### PR TITLE
Handle issues in a better way

### PR DESCRIPTION
**Pull request by bissenbay**
Related-To: https://github.com/thoth-station/cve-update-job/issues/218

### CODE REVIEW

File: thoth/storages/graph/postgres.py [hunk 1]
```diff
@@ -4495,7 +4495,7 @@ def sync_package_analysis_result(self, document: dict) -> None:
             self._session.commit()
 
     def _get_or_create_python_package_index(
-        self, index_url: str, only_if_enabled: bool = True
+        self, index_url: str, only_if_enabled: bool = False
```
Comment (fridex) [inline thoth/storages/graph/postgres.py]: I meant replacing line:

```python
    index = self._session.query(PythonPackageIndex).filter_by(url=index_url).one()
```

with:

```python
    index = self._get_or_create_python_package_index(index_url, only_if_enabled=False)
```

in `create_python_cve_record` method.

**Comment by goern** [comment]
Will it fix 218?

**Comment by fridex** [comment]
> Will it fix 218?

Yes, CVEs will be synced into the database and the PyPI index will be disabled until we run init-job.

**Comment by goern** [comment]
Do we need a new release due to this fix?

**Pull request merged**